iml-services/iml-action-runner/src/main.rs

```diff
@@ -49,12 +47,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     .map(|x| warp::reply::json(&x))
     .with(log);
 
-    let addr = unsafe { NetUnixListener::from_raw_fd(3) };
-
-    let listener = UnixListener::try_from(addr)?
-        .incoming()
-        .inspect_ok(|_| tracing::debug!("Client connected"));
```

Comment:
If we still need the `inspect_ok` i'll add it back. Thoughts?

Reply:
Yeah... I meant to add that inside of `get_tcp_or_unix_listener` since it can be placed on the resulting stream.

Reply:
In that case I can add it in iml_util.